README.md

```diff
@@ -542,6 +542,40 @@ showCard(Pair('X', '♠'));
 // ! TypeError: ‘showCard’ expected a value of type (Pair Rank Suit) as its first argument; received Pair("X", "♠")
 ```
 
+#### `EnumType`
+
+`EnumType` is used to construct [enumerated types][5].
+
+To define an enumerated type one must provide:
+
+  - an array of values with distinct [`R.toString`][6] representations.
+
+```haskell
+EnumType :: [Any] -> Type
+```
+
+For example:
+
+```javascript
+//    TimeUnit :: Type
+const TimeUnit = $.EnumType(['milliseconds', 'seconds']);
+
+const def = $.create($.env.concat([TimeUnit, $.ValidDate, $.ValidNumber]));
+
+//    convertTo :: TimeUnit -> ValidDate -> ValidNumber
+const convertTo =
+def('convertTo',
+    {},
+    [TimeUnit, $.ValidDate, $.ValidNumber],
+    (unit, date) => date.valueOf() * (unit === 'milliseconds' ? 1000 : 1));
```

Comment:
There's a logic error here, I believe: `date.valueOf()` returns the time in milliseconds—rather than seconds—since the epoch, so we should divide by 1000 in the `'seconds'` case rather multiply by 1000 in the `'milliseconds'` case.

We could use `switch` (and increase the number of members) to make `TimeUnit`'s enumerated nature more apparent:

```javascript
function recur(unit, date) {
  switch (unit) {
    'milliseconds': return date.valueOf();
    'seconds':      return recur('milliseconds', date) / 1000;
    'minutes':      return recur('seconds', date) / 60;
    'hours':        return recur('minutes', date) / 60;
  }
}
```